main.py

```diff
@@ -51,112 +27,233 @@ def send_static(path):
-                     chartrepourl, serviceowner, serviceowneremail, serviceownerphone, 
-                     slackchannel, discordchannel, hipchatchannel, pagerdutyurl, pagerdutybusinessurl
-                     from dm.dm_componentitem where id = %s"""
-
-            params = (compitemid,)
+                        kind, buildid, buildurl, chart, operator, builddate, dockersha, gitcommit,
+                        gitrepo, gittag, giturl, chartversion, chartnamespace, dockertag, chartrepo,
+                        chartrepourl, serviceowner, serviceowneremail, serviceownerphone, 
+                        slackchannel, discordchannel, hipchatchannel, pagerdutyurl, pagerdutybusinessurl
+                        from dm.dm_componentitem where id = %s"""
+    
+            params = (str(compitemid),)
             cursor.execute(sql, params)
             result = cursor.fetchall()
             if (not result):
                 result = [OrderedDict([('compid', -1), ('id', compitemid), ('name', None), ('rollup', None), ('rollback', None), ('repositoryid', None),
-                                       ('target', None), ('xpos', None), ('ypos', None),  ('kind', None), ('buildid', None), ('buildurl', None),
-                                       ('chart', None), ('operator', None), ('builddate', None), ('dockersha', None), ('gitcommit', None),
-                                       ('gitrepo', None), ('gittag', None), ('giturl', None), ('chartversion', None), ('chartnamespace', None), ('dockertag', None), ('chartrepo', None),
-                                       ('chartrepourl', None), ('serviceowner', None), ('serviceowneremail', None), ('serviceownerphone', None),
-                                       ('slackchannel', None), ('discordchannel', None), ('hipchatchannel', None), ('pagerdutyurl', None), ('pagerdutybusinessurl', None)])]
+                                        ('target', None), ('xpos', None), ('ypos', None),  ('kind', None), ('buildid', None), ('buildurl', None),
+                                        ('chart', None), ('operator', None), ('builddate', None), ('dockersha', None), ('gitcommit', None),
+                                        ('gitrepo', None), ('gittag', None), ('giturl', None), ('chartversion', None), ('chartnamespace', None), ('dockertag', None), ('chartrepo', None),
+                                        ('chartrepourl', None), ('serviceowner', None), ('serviceowneremail', None), ('serviceownerphone', None),
+                                        ('slackchannel', None), ('discordchannel', None), ('hipchatchannel', None), ('pagerdutyurl', None), ('pagerdutybusinessurl', None)])]
             cursor.close()
             conn.close()
-            return result
-        except Exception as err:
-            print(err)
-            conn.rollback()
-            return ({"message": str(err)}), HTTPStatus.INTERNAL_SERVER_ERROR
+        return result[0]
```

Comment:
Shoud return result and not result[0] since we want all rows returned by to the client side

Reply:
Since we are fetching data by its Id there will only be one object present. But yes we can make it standard like in post request. Resolved.